README.md

```diff
@@ -40,20 +42,78 @@ react-native run-android
 *Note* You need to have either a device emulator open already or a physical device plugged into your computer - otherwise you'll need to run through Android Studio
 *Note* If Android studio prompts you to update Gradle Files, you should do it
 
-### Testing
+## Testing
 Unit tests utilize Jests' Snapshots, located in src/****/__tests__ directories.  Mocks are located under root /__mocks__ directory.
 To run the test suite, use `npm test`.  
 
 About Snapshots:  Facebook's native apps use a system called "snapshot testing":  a snapshot test system that renders UI components, takes a screenshot and subsequently compares a recorded screenshot with changes made by an engineer.  It is meant to make sure components don't change unexpectedly.  The first time a test is run, Jest will create a snapshot file that needs to be committed alongside code changes.  On subsequent test runs Jest will simply compare the rendered output with the previous snapshot. If they match, the test will pass. If they don't match, either the implementation has changed and the snapshot needs to be updated with `npm test -- -u TESTNAME` or it indicates there is a bug that needs to be fixed.
 
 Snapshots should be committed to the repo and any changes to snapshots should be reviewed along with their PRs.
 
-### Debugging 
+## Building For Release
+
+### Automated 
+Our build scripts are powered by [Fastlane](https://fastlane.tools/). 99% of the time you should use the automated scripts to build and push the Android and iOS apps to Google Beta and Testflight because they will automatically clean the projects and node modules and handle incrementing the build numbers.
+
+That being said, there are few things you will need to do in order to get your dev environment ready to use the build scripts.
+
+#### General Preperation:
+
+- Make sure Ruby is installed on your machine  
+- Install Fastlane using either  
+RubyGems: 
+	`sudo gem install fastlane -NV`  
+or Hombrew `brew cask install fastlane`
+
+
+#### iOS Preperations: 
+   Before getting everything setup it is probably worth spending some time understanding how [Fastlane match](https://codesigning.guide/) works. 
+
+- Make sure that you have access to the [private repo](https://github.com/zooniverse/mobile-provisioning-profiles) that houses our provisioning profiles.
```

Comment:
What is this? I don't think I have access.

Reply:
Check out https://codesigning.guide/ for an explanation. Basically this is a private repo that stores our provisioning profiles encrypted. And fastlane automatically pulls them down when you build so we only have to manage two provisioning profiles (developer and distribution) instead of two for each member of team. 

Basically this will make it easier in the future if we ever want to add a new member to the team or if any other developer wants to start making production builds